tests/coreTest.php

```diff
@@ -94,4 +95,24 @@ function testSiteSelectionViaCwd() {
     }
     chdir($cwd);
   }
+
+  function testOptionsUri() {
+    // Put a yml file in the drush folder.
+    $drush_config_file = Path::join($this->getSut(), 'drush', 'drush.yml');
+    $test_uri = 'http://test.uri';
+    $options_with_uri = [
+      'options' => [
+        'uri' => $test_uri,
+      ],
+    ];
+    $options = [
+      'format' => 'json',
+    ];
+    file_put_contents($drush_config_file, Yaml::dump($options_with_uri, PHP_INT_MAX, 2));
+    $this->drush('core-status', [], $options);
```

Comment:
You typically have to specify --root option in order for a drush() call to work. This is the second issue where I have seen PRs omit that and seemingly the test passes. Not sure whats going on but I'd like to find out.